R/model_engines.R

```diff
@@ -235,7 +235,7 @@ add_multinom_reg_h2o <- function() {
     mode = "classification",
     value = list(
       interface = "data.frame",
-      protect = c("x", "y", "family"),
```

Comment:
Did you take family out on purpose?

Reply:
I think that's more consistent with all other models that uses `h2o.glm`. Or do you think we should protect all `family` in those?